libs/domain/content/src/models/content.model.ts

```diff
@@ -19,6 +19,7 @@ export interface ContentQualifier {
 
 export interface ContentField {
   [key: string]: unknown;
+  heading?: string;
```

Comment:
we won't need this with entity fields, right? can we leave it out already?

Reply:
I will fix it in model parsing